### PR TITLE
Add support for format `.drc`, Google's Draco 3D Data Compression

### PR DESCRIPTION
**Pull request by Borewit**
Add support for Google's Draco 3D Data Compression:
- extension: `.drc`
- MIME-type: `application/vnd.google.draco` (not a formal MIME type, neither a widely used convention) 

More info about this format:
- https://google.github.io/draco/

Related discussion to it's MIME-type:
- https://github.com/google/draco/issues/761

Resolves: #427

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -254,5 +255,6 @@
 	},
 	"ava": {
 		"serial": true
-	}
+	},
+	"packageManager": "yarn@1.22.22+sha512.a6b2f7906b721bba3d67d4aff083df04dad64c399707841b7acf00f6b133b7ac24255f2652fa22ae3534329dc6180534e98d17432037ff6fd140556e2bb3137e"
```
Comment (sindresorhus) [inline package.json:259]: Unrelated change.

Comment (Borewit) [reply package.json]: Sorry, super annoying, I think it is corepack is causing the unwanted change
